### ISSUE TITLE
why i got this error?

### ISSUE THREAD

**Issue by DiaoYuxian** [chenyuntc/simple-faster-rcnn-pytorch#109]
my env：
Python3.5.4
CUDA Version 8.0.61
Pytorch 1.0.1

python3 train.py train --env='fasterrcnn-caffe' --caffe-pretrain

/usr/local/lib/python3.5/site-packages/IPython/core/history.py:226: UserWarning: IPython History requires SQLite, your history will not be saved
  warn("IPython History requires SQLite, your history will not be saved")
Traceback (most recent call last):
  File "train.py", line 24, in <module>
    resource.setrlimit(resource.RLIMIT_NOFILE, (20480, rlimit[1]))
ValueError: current limit exceeds maximum limit

If you suspect this is an IPython bug, please report it at:
    https://github.com/ipython/ipython/issues
or send an email to the mailing list at ipython-dev@python.org

You can print a more detailed traceback right now with "%tb", or use "%debug"
to interactively debug it.

Extra-detailed tracebacks for bug-reporting purposes can be enabled via:
    %config Application.verbose_crash=True

**Comment by stillstream** [comment]
I have the same problem.Have you slove it?

**Comment by DiaoYuxian** [comment]
no, I haven't.  o(╥﹏╥)o    I gave it up，and tried another project.  https://github.com/jwyang/faster-rcnn.pytorch

**Comment by DiaoYuxian** [comment]
> I have the same problem.Have you slove it?
no, I haven't.  o(╥﹏╥)o    I gave it up，and tried another project.  https://github.com/jwyang/faster-rcnn.pytorch

**Comment by heekhero** [comment]
> > I have the same problem.Have you slove it?
> 
> no, I haven't. o(╥﹏╥)o I gave it up，and tried another project. https://github.com/jwyang/faster-rcnn.pytorch

Can you run the new project well? I chose this project because I couldn't run jwyang‘s project.

**Comment by CUGfred** [comment]
i am the same as heekhero,anybody know how to handle the problem?

**Comment by CUGfred** [comment]
i ve solved this problem(while i ve got another one after that),firstly
trian.py
import resource
rlimit = resource.getrlimit(resource.RLIMIT_NOFILE)
resource.setrlimit(resource.RLIMIT_NOFILE, (20480, rlimit[1]))
cause this problem,u should add a "print(rlimit)" after rlimit definition.
then,copy the content printed to the "(20480, rlimit[1])" and you will get another problem like me probably.

**Comment by cjnjuwhy** [comment]
I don't know the details, but the `resource.setrlimit(resource.RLIMIT_NOFILE, (20480, rlimit[1]))` should be like this: 20480 < rlimit[1]
so, you should print the rlimit before this line, and make sure rlimit[1] > 20480 (if not, change 20480 to 2048 or other number)
```python
rlimit = resource.getrlimit(resource.RLIMIT_NOFILE)

print(rlimit)
# rlimit = (1024, 4096) in my machine
resource.setrlimit(resource.RLIMIT_NOFILE, (2048, rlimit[1]))
```

**Comment by tkone2018** [comment]
resource only used in linuxs

**Comment by Xavier-cvpr** [comment]
@cjnjuwhy  you are right,I adopte  your method to solve the problem,thank you
